src/client/components/AddReferenceLayerModal.tsx

```diff
@@ -0,0 +1,475 @@
+                        <Box sx={style.customInputContainer}>
+                          <InputField
+                            field="name"
+                            label="Reference layer name"
+                            resource={createLayerResource}
+                            inputProps={{
+                              onChange: (e: React.ChangeEvent<HTMLInputElement>) => {
+                                const name = e.currentTarget.value;
+                                name !== null &&
+                                  setCreateLayerResource({
+                                    data: {
+                                      ...formData,
+                                      name
+                                    }
+                                  });
+                              }
+                            }}
+                          />
+                        </Box>
+                      </Flex>
+                    </Card>
+                    <Card sx={{ variant: "card.flat" }}>
+                      <Flex sx={{ flexWrap: "wrap" }}>
+                        <Box sx={style.customInputContainer}>
+                          <Label
+                            htmlFor="label-field-dropdown"
+                            sx={{ display: "inline-block", width: "auto", mb: 0, mr: 2 }}
+                          >
+                            Label field:
+                          </Label>
+                          <Select
+                            id="label-field-dropdown"
+                            value={createLayerResource.data.label_field || "Select label field..."}
+                            onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
+                              const label = e.currentTarget.value;
+                              setCreateLayerResource({
+                                data: { ...createLayerResource.data, label_field: label }
+                              });
+                            }}
+                            sx={{ width: "150px" }}
+                          >
+                            <option>Select label field...</option>
+                            {labelOptions}
+                          </Select>
+                        </Box>
+                      </Flex>
+                    </Card>
+                  </Box>
+                ) : (
+                  <Box></Box>
+                )}
+                <Button sx={{ variant: "buttons.linkStyle" }} onClick={() => resetForm()}>
+                  <Icon name="undo" />
+                  Redo
+                </Button>
+              </Box>
+            ) : (
+              <Box>
+                <Box>This will add a reference layer to your project.</Box>
+                <input
+                  onChange={event => event.target.files && setFile(event.target.files[0])}
+                  ref={fileInputRef}
+                  type="file"
+                  // accept={".csv" || ".geojson"}
```

Comment:
According to MDN this takes a comma separated list of values https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept
```suggestion
                  accept={".csv,.geojson"}
```